pyccl/covariances.py

```diff
@@ -315,30 +332,30 @@ def angular_cl_cov_SSC(cosmo, cltracer1, cltracer2, ell, tkka,
     cosmo_in = cosmo
     cosmo = cosmo.cosmo
 
-    if isinstance(tkka, Tk3D):
-        tsp = tkka.tsp
+    if isinstance(t_of_kk_a, Tk3D):
+        tsp = t_of_kk_a.tsp
     else:
-        raise ValueError("tkka must be a pyccl.Tk3D")
+        raise ValueError("t_of_kk_a must be of type pyccl.Tk3D")
 
     # Create tracer colections
     status = 0
     clt1, status = lib.cl_tracer_collection_t_new(status)
-    for t in cltracer1._trc:
+    for t in tracer1._trc:
         status = lib.add_cl_tracer_to_collection(clt1, t, status)
     clt2, status = lib.cl_tracer_collection_t_new(status)
-    for t in cltracer2._trc:
+    for t in tracer2._trc:
         status = lib.add_cl_tracer_to_collection(clt2, t, status)
-    if cltracer3 is None:
+    if tracer3 is None:
         clt3 = clt1
     else:
         clt3, status = lib.cl_tracer_collection_t_new(status)
-        for t in cltracer3._trc:
+        for t in tracer3._trc:
             status = lib.add_cl_tracer_to_collection(clt3, t, status)
-    if cltracer4 is None:
+    if tracer4 is None:
         clt4 = clt2
     else:
         clt4, status = lib.cl_tracer_collection_t_new(status)
-        for t in cltracer4._trc:
+        for t in tracer4._trc:
             status = lib.add_cl_tracer_to_collection(clt4, t, status)
```

Comment:
Like I mentioned earlier, shall we allow for `tracer2=None` and rename `tracer1` to `tracer`?